deploy/template.yaml

```diff
@@ -127,9 +133,10 @@ Resources:
       SourceSecurityGroupId: !GetAtt LoadBalancerSG.GroupId
 
   AccessLogsBucket:
+    Condition: IsNotDevelopment
     Type:
       AWS::S3::Bucket
-      #checkov:skip=CKV_AWS_18: This is the bucket where our access logs go and AWS advise not sending a bucket's access logs to itself.
+    #checkov:skip=CKV_AWS_18: This is the bucket where our access logs go and AWS advise not sending a bucket's access logs to itself.
```

Comment:
You might need to add a tag to skip this if you have the checkov hooks stack deployed in your account.
```Tags:
        - Key: CheckovRulesToSkip
          Value: CKV_AWS_18```